wallet/src/Pos/Util/Mnemonic.hs

```diff
@@ -94,15 +93,11 @@ data MnemonicErr
 mkEntropy
     :: forall n csz. (ValidEntropySize n, ValidChecksumSize n csz)
     => ByteString
-    -> Either MnemonicErr (Entropy n)
-mkEntropy =
-    let
-        n = fromIntegral $ natVal (Proxy @n)
-    in
-        maybe (Left $ MnemonicErrInvalidEntropyLength n) Right . toEntropy @n
+    -> Either (MnemonicErr csz) (Entropy n)
+mkEntropy = left MnemonicErrInvalidEntropy . toEntropy @n
```

Comment:
This can now simply be a re-export with a bit of renaming on top for consistency

```hs
mkEntropy
    :: forall n csz. (ValidEntropySize n, ValidChecksumSize n csz)
    => ByteString
    -> Either EntropyError (Entropy n)
mkEntropy = toEntropy @n
```